src/packagedcode/golang.py

```diff
@@ -70,10 +71,13 @@ class GolangPackage(models.Package):
 
     @classmethod
     def recognize(cls, location):
-        if fileutils.file_name(location).lower() == 'go.mod':
-            gomod_obj = GoMod()
-            gomod_data = gomod_obj.parse_gomod(location)
+        filename = fileutils.file_name(location).lower()
+        if filename == 'go.mod':
+            gomod_data = GoMod.parse_gomod(location)
```

Comment:
Please avoid `_data` suffix here, it does not bring anything to the reader IMHO

Reply:
:+1: